components/LeagueCard/LeagueCard.test.tsx

```diff
@@ -85,4 +86,27 @@ describe('LeagueCard with custom league logo', () => {
     expect(leagueCardTitle).toBeInTheDocument();
     expect(leagueCardTitle).toHaveTextContent('League 1');
   });
+
+  it('should apply hover effect on mouse hover', () => {
+    render(
+      <Link
```

Comment:
I missed this. You do not need to recall <Link> again. That's already instantiated inside <LeagueCard>. When you use the component, it comes with everything inside of it. Your screen to get the data-testid will still work because that same exact test id is in <LeagueCard>.

Reply:
Thank you for explaining and catching this! I modify the code.